<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextBrowser" name="messageTextBrowser">
      <property name="placeholderText">
       <string>历史聊天记录</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="messageTextEdit">
      <property name="placeholderText">
       <string>在此输入消息</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="sendMessageButton">
      <property name="text">
       <string>发送消息</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sendFileButton">
      <property name="text">
       <string>发送文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2" alignment="Qt::AlignRight">
     <widget class="QTableWidget" name="userTableWidget">
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>主机名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP地址</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="switchCastTypeButton">
      <property name="text">
       <string>切换消息发送方式（目前：局域网多播）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionport">
   <property name="text">
    <string>port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
